packages/ui/atomic/create-atomic-component/index.js

```diff
@@ -136,61 +136,60 @@ const ensureComponentValidity = (tag) => {
   }
 };
 
+const ensureDirectoryValidity = () => {
+  const cwdFiles = readdirSync(cwd(), {withFileTypes: true});
+  const hasPackageInCwd = cwdFiles.some(
+    (dirent) => dirent.name === 'package.json'
+  );
+  if (cwdFiles.length > 0 && !hasPackageInCwd) {
+    handleErrors(
+      new InvalidProjectDirectory(
+        'Current working directory is either not empty or not an npm project (no package.json found). Please try again in an empty directory.'
```

Comment:
```suggestion
        'Current working directory is either not empty or not an npm project (no package.json found). Please try again in a valid project'
```
maybe specifying how to initialize an atomic project

Reply:
Maybe link to some relevant doc which explains the process e.g https://docs.coveo.com/en/cli/ or something more specific if available

Reply:
Disagreeing here: The create-atomic-(result)-component packages are initializers that are agnostic of the concept of atomic project.
i.e. you can use them without an atomic project.

Reply:
I personally think it's valid to refer to it as a `valid project` as that isn't specific to an atomic project. I think I'll update the link for this description to instead point to the `atomic:cmp` command documentation as it clearly outlines in what contexts the command will work:
https://docs.coveo.com/en/cli/#coveo-atomiccmp-name